Bulky.DataAccess/Repository/Repository.cs

```diff
@@ -24,7 +24,7 @@ public void Add(T entity)
         {
             if (entity == null)
             {
-                throw new Exception("The Information Could Not Be Created");
+                throw new Exception("The Table Could Not Be Created");
```

Comment:
Its not a table being added though. In order to save time a word used more often would be "the item"